### PR TITLE
Update to support Django 1.11 and Python 3.6

### PR DESCRIPTION
**Pull request by dpoirier**
Drop support for Python 3.2 and 3.3, and unsupported versions
of Django.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,8 @@
     "codemirror": ">=5.10 <6.0",
     "jquery": ">=2.2 <2.3",
     "jshint": "^2.9.5",
+    "less": "^2.7.3",
+    "phantomjs-prebuilt": "^2.1.16",
```
Comment (ironfroggy) [inline package.json]: What changed that required these be added now, if they were already used before?

Comment (dpoirier) [reply package.json]: All I know is that it didn't work without them. Maybe previous testers have had less and phantomjs installed globally and I don't?

**Comment by ironfroggy** [comment]
The tests pass, but the example project doesn't seem to work. I can run it, but the demo/demo login doesn't work. No errors, just silently fails to login.

**Comment by dpoirier** [comment]
example/README.rst says to create the demo/demo superuser as part of setting up the example project. The sphinx docs refer to the example/README.rst.  Is there somewhere else we could mention this that people would be more likely to notice?

**Comment by mlavin** [comment]
Everything ran locally for me (tests and example project). Looks like there are lots of problems with Django 2.0 but this is the right first step.

**Comment by ironfroggy** [comment]
Great, I'll walk through the release process on Monday. Merging now.

**Pull request merged**